### DISCUSSION TITLE
feat: Prompt API Search (by tags)

### DISCUSSION THREAD

**Discussion by bryan-agicap** [langfuse/langfuse#1755] [Ideas]
### Describe the feature or potential improvement

Now that we have prompts tags, would be awesome to list/search prompts with API.

We start having a lot of prompts in langfuse, and we often need to make the user select a prompt in the frontend.

![image](https://github.com/langfuse/langfuse/assets/145330610/8b829677-8f1c-4c45-aa56-f2515584f6a3)

**Comment by marcklingen** [comment]
Tanks for sharing, fully agree that there needs to be a GET API to get all prompts.

Work in progress: https://github.com/langfuse/langfuse/pull/1402

**Comment by marcklingen** [comment]
Released in https://github.com/langfuse/langfuse/releases/tag/v2.37.0
